[Needs SC vote] Steering Committee: votes on new members to be conducted on the forum; change the chairperson election procedure (#1085)
ansible/ansible-documentation: docs/docsite/rst/community/steering/steering_committee_membership.rst

Process
^^^^^^^^

Any community member may nominate someone or themselves for Steering Committee membership.

The process to join the Steering Committee consists of the following steps:

> #. The nominator contacts one of the :ref:`current Committee members <steering_members>` or by sending an email to ``ansible-community@redhat.com``. This step does not apply when the nominator is a Committee member.

Reviewer: Maybe something like

```suggestion
#. The nominator contacts one of the :ref:`current Committee members <steering_members>` or by sending an email to ``ansible-community@redhat.com``. Existing members nominating would skip to the following step.
```

The process to join the Steering Committee consists of the following steps:

> #. Any community member may nominate someone or themselves for Committee membership by contacting one of the :ref:`current Committee members <steering_members>`) or by sending an email to ``ansible-community@redhat.com``.

Reviewer 2: What if a Committee member wants to nominate someone else?

Author: Good point, changed, PTAL

> #. A Committee member who receives the nomination must inform the Committee about it by forwarding the full nomination message in a dedicated ``message`` to the `SteeringCommittee <https://forum.ansible.com/g/SteeringCommittee>` group on the forum.

Reviewer 2: ```suggestion
#. A Committee member who receives the nomination must inform the Committee about it by forwarding the full nomination message in a private message to the `SteeringCommittee <https://forum.ansible.com/g/SteeringCommittee>` group on the forum.
```

#. Any community member may nominate someone or themselves for Committee membership by contacting one of the :ref:`current Committee members <steering_members>`) or by sending an email to ``ansible-community@redhat.com``.
#. A Committee member who receives the nomination must inform the Committee about it by forwarding the full message.

> #. The vote is conducted on the forum in a dedicated community topic visible only to the `SteeringCommittee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group. Nominees must receive a majority of votes from the present Committee members to be added to the Committee.

Reviewer 3: ```suggestion
#. The vote is conducted on the forum in a dedicated community topic visible only to the `Steering Committee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group. Nominees must receive a majority of votes from the present Committee members to be added to the Committee.
```

> #. The vote is conducted on the forum in a dedicated community topic visible only to the `Steering Committee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group. Nominees must receive a majority of votes from the present Committee members to be added to the Committee.

Reviewer 2: I guess this would have to be a private message to the SC group, not a topic. Also, I'd like to know whether new SC members could see the previous private messages from before they joined the group. If they can, we may need to send these messages to each individual member as opposed to the group as a whole.

Reviewer 4: > Also, I'd like to know whether new SC members could see the previous private messages from before they joined the group.

I think James confirmed that they can.

Reviewer 2: > > Also, I'd like to know whether new SC members could see the previous private messages from before they joined the group.
> 
> I think James confirmed that they can.

That was different, as I purposely sent the PM to each group member as an individual as opposed to the group.

Chairperson
------------

> The chairperson election happens once a year at the time the Committee agrees on.

Reviewer 5: This part is unclear to me. The committee will agree on a specific time of year that we will use every year? (in which case, that should be decided first and then put in this PR)

Or just whenever we feel like it we can have a new election? (in which case, it probably means we need to have a vote to have an election)

Author: @briantist i wouldn't specify a fixed time here because we can easily miss it, so added voting. PTAL

> The chairperson election happens once a year at the time the Committee agrees on by voting in a dedicated community topic.

Reviewer 2: Same issue here about the community topic vs. private forum message

If the current chairperson has to step down early, the election happens immediately.

The process of the election consists of the following steps:

> #. Send a dedicated ``message`` to the `Steering Committee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group.

Reviewer 2: ```suggestion
#. Send a private message to the `Steering Committee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group.
```

> #. Create a dedicated community topic visible only to the `SteeringCommittee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group.

Reviewer 3: ```suggestion
#. Create a dedicated community topic visible only to the `Steering Committee <https://forum.ansible.com/g/SteeringCommittee>`_ forum group.
```

Chairperson
------------

> The chairperson election happens once a year at the time the Committee agrees on by voting in a dedicated forum thread.

Reviewer: Don't think we've done this so far, do we need this line?

Author: @gundalow it's a good point worth discussing!
I've already started the vote though. I don't think this can be discussed quickly and i think it should be done thoroughly.
Let's maybe postpone this question to another pass?
